src/callbacks_step/analysis.jl

```diff
@@ -377,8 +377,15 @@ function (analysis_callback::AnalysisCallback)(io, du, u, u_ode, t, semi)
         println()
     end
 
-    # Calculate L2/Linf errors, which are also returned
-    l2_error, linf_error = calc_error_norms(u_ode, t, analyzer, semi, cache_analysis)
+    if :l2_error in analysis_errors || :linf_error in analysis_errors
+        # Calculate L2/Linf errors, which are also returned
+        l2_error, linf_error = calc_error_norms(u_ode, t, analyzer, semi,
+                                                cache_analysis)
+    else
+        # Names `l2_error`, `linf_error` need to be defined
+        l2_error = nothing
+        linf_error = nothing
```

Comment:
No, I mean `return nothing` and change the line you refer to to

```julia
analysis_callback(io, du, u, u_ode, t, semi)
```

since the return values `l2_error, linf_error` do not seem to be used anywhere.

Reply:
Sorry, I don't understand: The error values are passed on:

https://github.com/trixi-framework/Trixi.jl/blob/17d507071622a121afeaee810618ed0dc02bb5dd/src/callbacks_step/analysis.jl#L335-L358

And I am not sure if we can change the code such that we can return from line 335

Reply:
:bulb: